yolo/exporter.py

```diff
@@ -61,6 +61,7 @@ def export_openvino(self, version):
         f"--model_name '{self.model_name}' " \
         '--data_type FP16 ' \
         '--reverse_input_channels ' \
+        '--use_legacy_frontend ' \
```

Comment:
Can we make this togglable under advanced options for both exporters?

Reply:
So you mean like option (checkbox) for using the using the legacy flag?

Reply:
Yeah exactly.